pipelines/rbsa/tasks/find_sensitivities.py

```diff
@@ -0,0 +1,214 @@
+        self.input_artifact_sensitivity_temperatures = 'SENSITIVITY_TEMPERATURES.json'
+        self.output_artifact_loadshapes = 'loadshapes.csv'
+        self.my_data_files = [
+            { 'name': self.input_artifact_normal_loads, 'read_type': SupportedFileReadType.DATA },
+            { 'name': self.input_artifact_sensitivity_temperatures, 'read_type': SupportedFileReadType.CONFIG },
+            { 'name': 'noaa/594.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/596.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/597.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/598.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/833.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/835.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/836.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/837.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/838.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/970.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/971.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/972.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/973.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/974.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/980.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/981.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/982.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/983.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/984.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/985.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/988.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/989.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/990.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/991.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/992.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'noaa/993.csv', 'read_type': SupportedFileReadType.DATA },
+        ]
+        self.task_function = self._task
+
+    def _get_data(self):
+        return self.load_data(self.my_data_files)
+        
+    def _task(self):
+        data_map = self._get_data()
+        
+        self.sensitivity_temperatures = data_map[self.input_artifact_sensitivity_temperatures]['residential']
+
+        self.theat = self.sensitivity_temperatures['theat']
+        self.tcool = self.sensitivity_temperatures['tcool']
+
+        print(self.theat)
```

Comment:
If this is needed, make it a logging statement.

Reply:
not needed